scripts/build-databroker.sh

```diff
@@ -112,12 +123,13 @@ function build_target() {
     #
     # this is solved by using different target-dirs for each platform
     echo "Building databroker for target $target_rust"
-    cross build --target $target_rust --target-dir ./target-$target_docker --features $KUKSA_DATABROKER_FEATURES --bin databroker --release
+    env
+    cross build --target $target_rust --target-dir ./target-$target_docker --features $KUKSA_DATABROKER_FEATURES --bin databroker --profile ${KUKSA_DATABROKER_PROFILE}
```

Comment:
Is `--release` and `--profile release` equivalent ways to express the same thing?

Reply:
have only tried `--profile release` yet